### PR TITLE
Added all Sort commands

### PR DESCRIPTION
**Pull request by SpecOps2016**

### CODE REVIEW

**Review by Fuzanwenn — APPROVED** [review]
LGTM!

**Review by TZL0 — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/Main.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * This is a workaround for the following error when MainApp is made the
  * entry point of the application:
  *
- *     Error: JavaFX runtime components are missing, and are required to run this application
+ *     Error: JavaFX runtime com.ponents are missing, and are required to run this application
```
Comment (TZL0) [inline src/main/java/seedu/address/Main.java:11]: Wrong spelling?

**Review by TZL0 — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/address/Main.java (hunk 1 above)
Comment (SpecOps2016) [reply src/main/java/seedu/address/Main.java]: Resolved

**Review by TZL0 — APPROVED** [review]

**Pull request merged**